frontend/packages/core/src/typography.tsx

```diff
@@ -118,25 +118,26 @@ type TextVariant =
   | "overline"
   | "input";
 
-const StyledTypography = styled.div<{ variant: TextVariant }>(
-  {
-    color: "#0D1030",
-  },
-  props => ({
-    fontSize: `${STYLE_MAP[props?.variant || 0].size}px`,
-    fontWeight: STYLE_MAP[props?.variant || 0].weight,
-    lineHeight: `${STYLE_MAP[props?.variant || 0].lineHeight}px`,
-    ...STYLE_MAP[props?.variant || 0].props,
-  })
-);
+const StyledTypography = styled.div<{ variant: TextVariant }>(props => ({
+  color: props["data-color"],
+  fontSize: `${STYLE_MAP[props?.variant || 0].size}px`,
+  fontWeight: STYLE_MAP[props?.variant || 0].weight,
+  lineHeight: `${STYLE_MAP[props?.variant || 0].lineHeight}px`,
+  ...STYLE_MAP[props?.variant || 0].props,
+}));
 
 export interface TypographyProps {
   variant: TextVariant;
   children: React.ReactNode;
+  color?: string;
 }
 
-const Typography = ({ variant, children }: TypographyProps) => {
-  return <StyledTypography variant={variant}>{children}</StyledTypography>;
+const Typography = ({ variant, children, color = "#0D1030" }: TypographyProps) => {
```

Comment:
i could have just inline styled the font color in the import component but I think users would want to be able to change the font color, so I added a color prop